Trust/Trust/Main/Home/Settings/PreferencesView.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  PreferencesView.swift
+//  Trust
+//
+//  Created by Vilma Valiente on 6/8/24.
+//
+
+import SwiftUI
+
+struct PreferencesView: View {
+    // MARK: - Public Properties
+
+    var navigator: Navigator
+
+    // MARK: - Private Properties
+
+    private let navigationTitle = LocalizedStringKey("preferencesView_navigationTitle")
+
+    // MARK: - Body
+
+    var body: some View {
+        VStack {
+            Text(verbatim: "Preferences")
```

Comment:
Please excuse the emptiness of the view. The main focus of this PR is the navigation.